warehouse/query-core/src/main/java/datawave/query/iterator/profile/QuerySpan.java

```diff
@@ -28,6 +28,8 @@
     
     protected long seek = 0;
     
+    protected long yield = 0;
```

Comment:
This should probably be a boolean.  Either we yielded or we didn't

Reply:
Updated